src/functions.php

```diff
@@ -2,9 +2,72 @@
 
 namespace Amp\Http\Server;
 
+use Amp\ByteStream\InputStream;
 use Amp\Http\Status;
+use Amp\Promise;
+use Amp\Socket\Server as Socket;
+use Psr\Log\LoggerInterface;
+use Psr\Log\NullLogger;
+use function Amp\call;
 
 function redirectTo(string $uri, int $statusCode = Status::FOUND): Response
 {
     return new Response($statusCode, ['location' => $uri]);
 }
+
+/**
+ * Returns a response with the html content type.
+ *
+ * @param InputStream|string|null $content
+ * @param int $status
+ * @return Response
+ */
+function html($content, int $status = Status::OK): Response
+{
+    return new Response($status, [
+        'Content-Type' => 'text/html; charset=utf-8'
+    ], $content);
+}
+
+/**
+ * Creates an HTTP server.
+ *
+ * This function is inspired in Golang's http module one. It provides sensible defaults for a really quick start
+ * and it is designed to cover 90% of use cases.
```

Comment:
It won't cover 90% I guess. It only listens with amphp/socket, while generally amphp/cluster will be used to allow for multi processing with multiple CPU cores.

Reply:
Yeah, well, we usually have a single core in our Kubernetes pods and we scale spawning more processes. But I see what you mean.

Willing to change the description though. 😛